src/panels/config/info/integrations-card.ts

```diff
@@ -128,10 +152,27 @@ class IntegrationsCard extends LitElement {
                       : html`
                           <td>${docLink}</td>
                           <td>${issueLink}</td>
-                          <td class="setup">
-                            ${setupSeconds ? html`${setupSeconds} s` : ""}
-                          </td>
                         `}
+                    <td class="log">
+                      ${logLevel === 10
+                        ? html`<mwc-button
```

Comment:
Can we make this a toggle instead of a button?

Reply:
I'm going to reset and start over with a new design after everything is mopped up from the 2022.2 release.  I'll send some mock ups once I have cycled back to this. 

Thanks!